toolchain/check/import_ref.cpp

```diff
@@ -888,6 +888,34 @@ class ImportRefResolver {
     return context_.inst_blocks().Add(new_patterns);
   }
 
+  // Returns a version of import_return_slot_pattern_id localized to the current
+  // IR.
+  auto GetLocalReturnSlotPatternId(SemIR::InstId import_return_slot_pattern_id)
+      -> SemIR::InstId {
+    if (!import_return_slot_pattern_id.is_valid()) {
+      return import_return_slot_pattern_id;
```

Comment:
```suggestion
      return SemIR::InstId::Invalid;
```

Suggesting to use an explicit value return when equivalent, since it avoids a small bit of reasoning about what the value should be.

In this file, there's also consistency (maybe partly because I wrote most of this): `ResolveType` does similar to what you're doing here, but other functions are returning `Invalid` explicitly, including `GetLocalInstBlockId`, `MakeIncompleteGeneric`, `GetOrAddLocalSpecific` and `ResolveLocalInstBlock`.